mne/time_frequency/tfr.py

```diff
@@ -1511,9 +1511,34 @@ def plot_joint(self, timefreqs=None, picks=None, baseline=None,
             fmin = freq - freq_half_range
             fmax = freq + freq_half_range
 
+            data = tfr.data
+
+            if layout is None:
+                from ..channels.layout import find_layout
+                pos = find_layout(tfr.info).pos
+            else:
+                pos = layout.pos
+
+            # merging grads here before rescaling makes ERDs visible
+            if ch_type == 'grad':
+                from ..channels.layout import find_layout
+                from ..channels.layout import _merge_grad_data
+                from ..channels.layout import _pair_grad_sensors
```

Comment:
these should be put together in a single call like
```
from ..channels.layout import (find_layout, _merge_grad_data,
                               _pair_grad_sensors)
```
It makes it more immediately obvious that they are all from the same place (and probably takes one fewer line to do it).

You can also put this as the first line within the function, then you don't need the one up on 1517.